anta/custom_types.py

```diff
@@ -4,9 +4,9 @@
 """Module that provides predefined types for AntaTest.Input instances."""
 
 import re
-from typing import Annotated, Literal
+from typing import Annotated, Literal, Self, get_args
```

Comment:
`Self` was added in 3.11: https://docs.python.org/3/library/typing.html#typing.Self

Reply:
I am crying